src/sagas/wallet.js

```diff
@@ -380,7 +380,7 @@ export function* loadTokens() {
   //
   // Note: We need to download the balance of all the tokens from the wallet so we can
   // hide zero-balance tokens
-  for (const token of allTokens) {
+  for (const token of Object.values(allTokens)) {
```

Comment:
I know allTokens key and value are the same, but I think we should have a consistent usage, maybe change this to `Object.keys`